### DISCUSSION TITLE
Variable:it take too long time when update the variable select option

### DISCUSSION THREAD

**Discussion by xingchensly** [grafana/grafana#38144] [Dashboards, Panels & Visualizations]
![image](https://user-images.githubusercontent.com/30857850/91517519-82266100-e920-11ea-93b0-3bbcb7cea5c7.png)

**What happened**:
as display above picture.  when i want to change option of the region at the top-left corner. such as from 1 to 3,  it take a long time which look like is the Request time of cvm in Network. about 1.63s , before the view updated(the option i selected update to the region view)
**What you expected to happen**:
it should update the region variable view right now , the Request should not block this render
**How to reproduce it (as minimally and precisely as possible)**:
1. if privateIp variable could not update when region change
2. The metricFindQuery in datasource has return a pending promise soon when it triggered
**Anything else we need to know?**:
the options of privateIp variable need to update data from the Request cvm, when the region has changed
**Environment**:
- Grafana version:7.0.3
- Data source type & version:own datasource
- OS Grafana is installed on:macOS Catalina 10.15.5
- User OS & Browser:macOS Chrome/84.0.4147.135 
- Grafana plugins:own plugin
- Others:

**Comment by torkelo** [comment]
Sounds like you just want the dropdown to show the value clicked on and then a loading spinner to indicate the next variable is now loading?

**Comment by xingchensly** [comment]
> Sounds like you just want the dropdown to show the value clicked on and then a loading spinner to indicate the next variable is now loading?

yes, when I clicked, just hope to show the clicked value immediately instead that waiting for the next variable Request return in Network

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
